core/modules/system/views/system.views.inc

```diff
@@ -120,6 +120,17 @@ function system_views_data() {
     ),
   );
 
+  // - info
+  $data['system']['system_info'] = array(
+    'title' => t('Module/Theme/Theme engine settings'),
```

Comment:
```suggestion
    'title' => t('Project/Theme engine properties'),
```

Reply:
Since other system handlers use `Module/Theme/Theme engine` in their names, I think we should either modify all of them, or none. I've chosen to keep the current name. To change the other ones we would need to open another issue.